compose.yaml

```diff
@@ -6,11 +6,25 @@ services:
       - 443:443
     depends_on:
       - tileserver
+    volumes:
+      - ./nginx/conf.d/:/etc/nginx/conf.d/
+      - ./nginx/cron/:/etc/periodic/weekly/
+      - ./nginx/letsencrypt/:/etc/letsencrypt/
+      - ./nginx/logs/nginx/:/var/log/nginx/
+      - ./nginx/logs/letsencrypt:/var/log/letsencrypt/
+  osmium:
+    build: osmium/.
+    volumes:
+      - ./data:/data
+  planetiler:
+    build: planetiler/.
+    environment:
+      JAVA_TOOL_OPTIONS: "-Xmx5g"
```

Comment:
This looks familiar but can you remind me what this does and why it's necessary?

Reply:
It [specifies the memory limit for the JVM](https://stackoverflow.com/a/14763095). The [planetiler instructions](https://github.com/onthegomap/planetiler?tab=readme-ov-file#usage) use 1g as the limit. Our input data is only 2gb; we could get away with keeping the memory limit at 1gb. But, the build is much faster with 5gb of memory allocated (~5mins v ~20mins)